scipy/stats/tests/test_distributions.py

```diff
@@ -1466,7 +1467,142 @@ def test_sf_isf(self):
+        betas = data[:,(3,)]
+        
+        tests = [
+            ['quadrature', None, 8],
+            ['zolotarev', None, 8],
+            ['fft', 0, 4]
+        ]
+        for default_method, fft_min_points, decimal_places in tests:
+            stats.levy_stable.pdf_default_method = default_method
+            stats.levy_stable.pdf_fft_min_points_threshold = fft_min_points        
+            pdf = stats.levy_stable.pdf(xs, alphas, betas, scale=1, loc=0)
+            assert_almost_equal(pdf, density, decimal_places, default_method)           
+            
+    def test_cdf_nolan_samples(self):
+        """ test pdf/cdf values against Nolan's stablec.exe output
+            see - http://fs2.american.edu/jpnolan/www/stable/stable.html
+            
+            stablec <<
+            2 # cdf
+            1 # Nolan S equivalent to S0 in scipy
+            1.25,1.75,.25 # alpha
+            -1,1,.5 # beta
+            -10,10,1 # x
+            1,0 # gamma, delta
+            2 # output file            
+        """        
+        data = np.load(os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                                 'data/stable-cdf-sample-data.npy')))
+        xs = data[:,(0,)]
+        cdf_test = data[:,(1,)]
+        alphas = data[:,(2,)]
+        betas = data[:,(3,)]
+        
+        tests = [
+            ['zolotarev', None, 8],
+            ['fft', 0, 2]
+        ]
+        for default_method, fft_min_points, decimal_places in tests:
+            stats.levy_stable.pdf_default_method = default_method
+            stats.levy_stable.pdf_fft_min_points_threshold = fft_min_points        
+            pdf = stats.levy_stable.cdf(xs, alphas, betas, scale=1, loc=0)
+            assert_almost_equal(pdf, cdf_test, decimal_places, default_method)  
+
+    def test_pdf_alpha_equals_one_beta_non_zero(self):
+        """ sample points extracted from Tables and Graphs of Stable Probability
+            Density Functions - Donald R Holt - 1973 - p 187. 
+        """
+        xs = np.array([0, 0, 0, 0, 
+                       1, 1, 1, 1, 
+                       2, 2, 2, 2, 
+                       3, 3, 3, 3, 
+                       4, 4, 4, 4])
+        density = np.array([.3183, .3096, .2925, .2622,
+                            .1591, .1587, .1599, .1635,
+                            .0637, .0729, .0812, .0955,
+                            .0318, .0390, .0458, .0586,
+                            .0187, .0236, .0285, .0384])
+        betas = np.array([0, .25, .5, 1, 
+                          0, .25, .5, 1, 
+                          0, .25, .5, 1, 
+                          0, .25, .5, 1, 
+                          0, .25, .5, 1])
+        
+        tests = [
+            ['quadrature', None, 4],
+            ['zolotarev', None, 1],
+        ]
+        
+        with np.warnings.catch_warnings():
+            np.warnings.filterwarnings('ignore', r'Density calculation unstable') 
```

Comment:
This causes the one test failure in the last CI run. Use `suppress_warnings` instead (check example from another test if you search for that name).